ls1intum/artemis-android
# `Bugfix`: Fix websocket conversation overview

### Problem Description
Conversation updates where not displayed via the websocket in the app. For example updating a conversation name did not reflect in the app. Also the unread messages counter did not increase for new incoming messages.

This is a follow up to #146.

### Changes
 
- Fix serialization of conversation updates -> updates such as renaming do now reflect in the app
- Also subscribed to postUpdate websocket to increment unread counters


### Steps for testing
Open a course both in the web app and go to the conversation overview in the android app:
- Web: Create new group chat with android user -> Android: see that group chat instantly appears 
- Web: Rename group chat -> Android: see that renaming is reflected without a manual refresh needed
- Web: Type a new message in the chat -> Android: see unread counter increases for chat
